server/routes/hdc.js

```diff
@@ -319,11 +305,49 @@ module.exports = function(
         res.redirect(`${nextPath}${nomisId}`);
     }));
 
-    router.post('/:sectionName/:formName/:nomisId', asyncMiddleware(async (req, res) => {
+    router.get('/:sectionName/:formName/:path/:nomisId', checkLicence, (req, res) => {
+        const {sectionName, formName, path, nomisId} = req.params;
+        logger.debug(`GET ${sectionName}/${formName}/${path}/${nomisId}`);
+
+        return formGet(req, res, sectionName, formName, nomisId);
```

Comment:
I don't understand why we have split out this function and then not done anything with the new variable (path) except log it. What is the gain?